FEATURES.md

```diff
@@ -156,6 +156,7 @@ Legend:
 |Window maximization toggle       |✔️     |✔️     |✔️         |✔️             |**N/A**|**N/A**|**N/A**   |
 |Fullscreen                       |✔️     |✔️     |✔️         |✔️             |**N/A**|**N/A**|❌        |
 |Fullscreen toggle                |✔️     |✔️     |✔️         |✔️             |**N/A**|**N/A**|❌        |
+|Video mode query                 |✔️     |✔️     |✔️         |✔️             |❌    |✔️     |❌        |
```

Comment:
Could you add a brief blurb describing this feature further up in the document, for the sake of consistency with the rest of the features?

Reply:
Done! I also noticed that this should probably be under the system information section rather than under windowing, so I've moved it there.